packages/@okta/vuepress-site/docs/concepts/identity-providers/index.md

```diff
@@ -121,9 +126,9 @@ You can define logic to determine which IdP individual end users are routed to w
 
 The Okta Sign-In Widget (version 2.5.0 or above) interacts with the IdP Discovery policy to redirect end users to the IdP determined by the policy.
 
-> **Note:** v1 of the Okta API supports IdP Discovery with the Okta-hosted Sign-In Widget only.
+> **Note:** Version 1 of the Okta API supports IdP Discovery with the Okta-hosted Widget only.
 
-If you don't use the Okta Sign-In Widget, you can implement a sign-in flow that uses the [Okta Authentication API](/docs/reference/api/authn/) and Okta [WebFinger](/docs/reference/api/webfinger/) endpoint. Use the Okta authentication API to implement sign-in yourself and integrate IdP Discovery by using the WebFinger endpoint. The WebFinger endpoint returns the name of the IdP that should be used for a given end user. Your org's defined IdP discovery policy determines the IdP that's returned.
+Orgs that interact directly with the [Okta Authentication API](/docs/reference/api/authn/) to implement the user sign-in flow rather than using the Widget, can still integrate IdP Discovery. Integrate IdP Discovery into your sigh-in flow by including a call to the Okta [WebFinger](/docs/reference/api/webfinger/) endpoint. This endpoint returns the IdP name that should be used for a given user, as determined by the org's defined IdP Discovery Policy.
```

Comment:
```suggestion
Orgs that interact directly with the [Okta Authentication API](/docs/reference/api/authn/) to implement the user sign-in flow rather than using the widget, can still integrate IdP Discovery. Integrate IdP Discovery into your sigh-in flow by including a call to the Okta [WebFinger](/docs/reference/api/webfinger/) endpoint. This endpoint returns the IdP name that should be used for a given user, as determined by the org's defined IdP Discovery Policy.
```

Reply:
Is widget capitalized when it's without "Sign-In"?